src/Elmish.WPF.Tests/ViewModelTests.fs

```diff
@@ -526,7 +526,7 @@ module OneWaySeqLazy =
     }
 
 
-  [<Fact>]
+  [<Fact>] // TODO: this test should pass, but it doesn't. Need to merge PR 478 first.
```

Comment:
Like the comment says, PR #478 needs to merge first so this test will pass

Reply:
This makes sense. So basically, by adding the lazy effect, it was calling "Get" once for each side of the equality check. Afer PR #478, it has the getter right there to call (which doesn't affect this call).

Reply:
> [...] by adding the lazy effect, it was calling "Get" once for each side of the equality check. Afer PR #478, it has the getter right there to call (which doesn't affect this call).

Yes, the `Get` was composed into both "sides" of the equality.  That is where two calls came from. The third call was in the code that prepares to call merge.  This is because the model mapping in the Lazy case mapped both the equals function and the nested binding.

Now this third call won't happen because the model that the merge preparation code needs is now passed down to it from the Lazy case.